wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -112,7 +112,7 @@ instance HasCoreConstants => MonadAddresses Pos.Wallet.Web.Mode.WalletWebMode wh
     type AddrData Pos.Wallet.Web.Mode.WalletWebMode = (AccountId, PassPhrase)
     getNewAddress (accId, passphrase) = do
         clientAddress <- L.newAddress RandomSeed passphrase accId
-        (, Nothing) <$> decodeCTypeOrFail (cadId clientAddress)
+        decodeCTypeOrFail (cadId clientAddress)
```

Comment:
In CSL-1483 I introduced a hack: `getNewAddresses` started returning not only an address, but also an optional `StakeholderId` who receives the stake. Now it's useless, because stake distribution is a part of address.